.github/workflows/finalize-release.yml

```diff
@@ -99,10 +99,21 @@ jobs:
           next_version=$(python scripts/release/next-version.py --component minor --pre-release dev -- $version)
           echo "NEXT_VERSION=$next_version" >> "$GITHUB_ENV"
         working-directory: tooling 
+      
+      - name: Generate token
+        if: env.HOTFIX_RELEASE == 'false'
+        id: generate-token
+        uses: actions/create-github-app-token@eaddb9eb7e4226c68cf4b39f167c83e5bd132b3e
```

Comment:
Could this use a tag instead of a commit SHA? This currently points to v1.5.1.

Reply:
No, because tags can be changed afterwards impacting our supply chain.

Reply:
I understand the justification but don't think that's a concern with GitHub-maintained and administered actions in the actions enterprise. It works either way though. Resolving this conversation (will follow-up async).